packages/devtools_app/lib/src/charts/treemap.dart

```diff
@@ -18,28 +16,28 @@ enum PivotType { pivotByMiddle, pivotBySize }
 class Treemap extends StatefulWidget {
   // TODO(peterdjlee): Consider auto-expanding rootNode named 'src'.
   const Treemap.fromRoot({
-    @required this.rootNode,
+    required this.rootNode,
     this.nodes,
-    @required this.levelsVisible,
+    required this.levelsVisible,
     this.isOutermostLevel = false,
-    @required this.width,
-    @required this.height,
-    @required this.onRootChangedCallback,
+    required this.width,
+    required this.height,
+    required this.onRootChangedCallback,
   });
```

Comment:
we should add an assert for both of these constructors that `rootNode == null != nodes == null`

Reply:
done